### ISSUE TITLE
Support for Imperial Units

### ISSUE THREAD

**Issue by kelvinfan001** [kelvinfan001/findalicious#93]
**Is your feature request related to a problem? Please describe.**
I'm American and I only measure in miles.

**Describe the solution you'd like**
I want support for miles instead of that weird KM thingy

**Describe alternatives you've considered**
I've tried converting, but 1.6 isn't a nice number to divide by.

**Additional context**
The imperial system is superior.

**Comment by avatarneil** [comment]
Boooo, miles are lame :P Heh, happy to take this on if you wanna assign to me ^_^

**Comment by kelvinfan001** [comment]
> Boooo, miles are lame

Agreed, but I wanted to be accommodating to my friendly southern neighbours. 
Please, feel free to tackle any of the issues here. Just remember to assign yourself so I don't accidentally create any chaos :)

**Comment by avatarneil** [comment]
> Just remember to assign yourself so I don't accidentally create any chaos :)

Hmm, I don't think I have permissions to assign myself to issues... Any chance you could add them @kelvinfan001?

**Comment by kelvinfan001** [comment]
I sent you an invite to be "collaborator". https://github.com/kelvinfan001/findalicious/invitations
lmk if there are any issues.
